_pages/federal-fieldguide/planning.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Federal field guide
+permalink: /federal-field-guide/
+sidenav: federal
+sticky_sidenav: true
+---
+
+# De-risking custom software projects: A field guide for federal agencies
```

Comment:
For now could you change this to just "Federal Field Guide"? That'll be consistent with the State one, and I am thinking the whole site is the really the first portion.